wordpress-mobile/WordPress-iOS
# Remove BottomSheet from WPiOS and move it to WPUI

This PR extracts the Bottom Sheet from this project and move it to WordPressUI.

### To test

1. Create a post
2. Tap Publish
3. Play with the Nudges view and make sure it behaves correctly
